docs/static/_redirects

```diff
@@ -14,6 +14,7 @@
 /guides/migration-v0.x /guides/migration-v0x/ 301
 /size-snapshot https://s3.eu-central-1.amazonaws.com/eps1lon-material-ui/artifacts/next/latest/size-snapshot.json 200
 /customization/css-in-js/ /css-in-js/basics/ 301
+/bug https://github.com/mui-org/material-ui/issues/new?template=1.bug.md 200
```

Comment:
301?

Reply:
I just copied the size-snapshot. I have no idea when 301 is more appropriate than 200.